x-pack/test/performance/services/performance.ts

```diff
@@ -166,6 +166,8 @@ export class PerformanceTestingService extends FtrService {
 
   private async tearDown(page: Page, client: CDPSession, context: BrowserContext) {
     if (page) {
+      // Journey is delayed to finish to guarantee all events reported properly to APM.
+      await page.waitForTimeout(5000);
       apm.flush();
       await client.detach();
```

Comment:
@vigneshshanmugam suggested to add 
`await page.waitForResponse((response) => response.url().includes('/rum/events'));` after flush() to make sure the RUM agent response is sent to APM server before browser is killed.
https://elastic.slack.com/archives/C017DFNCV5H/p1649889685672899?thread_ts=1649869787.688239&cid=C017DFNCV5H